ledger/README.md

```diff
@@ -61,6 +61,10 @@ interface:
   for recent blocks, if the tracker's state depends only on recent blocks
   (e.g., for the tracker that keeps track of the recently committed
   transactions).
+  `loadFromDisk` is expected to reconstruct a tracker from scratch with one
+  exception: `eternalData` might be survive between `loadFromDisk` calls
+  without being loaded from disk that is especially actual for non-persisting
```

Comment:
What do you mean by "that is especially actual for non-persisting trackers"?

Reply:
feel free to re-word. Other trackers might restore state from DB but bulletin - can't

Reply:
Ah, maybe "this is especially useful for trackers that don't rely on the tracker DB?" 
Feel free to write something else, I was confused by the word actual but it might just be me.